src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        return BallColors.values()[new Random().nextInt(BallColors.values().length)].toString();
```

Comment:
create field for Random inside the class

Reply:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery
Don’t create temporary variables if you can directly return value from the method
Bad example:

public User createUser(String name, int age) {
    User user = new User(name, age);
    return user;
}
Improved example:

public User createUser(String name, int age) {
    return new User(name, age);
}

Reply:
Random is not a temporary variable.
for each invocation of the method in ColorSupplier, there is going to be created new instance of Random. Which is not efficient. If you create a field in class and make it static. There is going to be just 1 object for all ColorSuplliers and all invocation of the method getRandomColor()
`private static final Random random = new Random()`

Reply:
static field was extracted